.github/workflows/release.yaml

```diff
@@ -28,19 +20,12 @@ jobs:
 
       - run: yarn install --frozen-lockfile
 
-      - name: Bump version
-        run: |
-          git config --global user.email "j@arcx.money"
-          git config --global user.name "ARCx"
-          npm version ${{ github.event.inputs.version }}
-
       - name: Push latest vestion
         run: git push origin main --follow-tags
 
-      - name: Release package
+      - name: Semantic Release
         run: |
           yarn build
-          npm config set //registry.npmjs.org/:_authToken ${NPM_TOKEN}
-          npm publish
+          yarn semantic-release
         env:
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
```

Comment:
We should remove NPM_TOKEN, once it works

Reply:
agreed